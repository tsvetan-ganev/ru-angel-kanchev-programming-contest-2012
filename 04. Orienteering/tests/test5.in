9 8
0107Ivanova w 08:27:56 08:38:59
0703Angelov m 10:08:08 10:25:20
0406Velichkov m 08:08:18 08:28:28
0802Atanasov m 08:19:45 08:39:55
0307Ivanov m 10:27:44 10:41:44
0208Kanev m 10:46:50 11:07:00
0901Atanasova w 08:05:10 08:18:22
0505Donevska w 09:29:58 09:42:35
0604Pavlov m 09:35:20 09:58:26