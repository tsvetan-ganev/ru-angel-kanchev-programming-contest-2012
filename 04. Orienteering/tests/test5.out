03 07 02 04 08 
01 05 09 
07 